test/suite/Amqp/Rpc/AmqpRpcServerTest.php

```diff
@@ -469,6 +471,169 @@ function () {
         }
     }
 
+    public function testReceiveRequestWithCustomErrorHandler()
```

Comment:
Assuming this test is largely a duplicate of an existing one, it could be pared down to just test for the specific differences when an exception handler is set.